errors.go

```diff
@@ -45,29 +59,30 @@ type ExitCoder interface {
 	ExitCode() int
 }
 
-// ExitError fulfills both the builtin `error` interface and `ExitCoder`
-type ExitError struct {
+type exitError struct {
```

Comment:
I think that this change is made because these functions are primarily for internal use and I have currently not seen an example where this function would be used by a user of this library